cli/stream_command.go

```diff
@@ -141,9 +141,9 @@ func configureStreamCommand(app commandHost) {
 		f.Flag("dupe-window", "Window size for duplicate tracking").Default("").StringVar(&c.dupeWindow)
 		f.Flag("json", "Produce JSON output").Short('j').BoolVar(&c.json)
 		f.Flag("max-age", "Maximum age of messages to keep").Default("").StringVar(&c.maxAgeLimit)
-		f.Flag("max-bytes", "Maximum bytes to keep").Int64Var(&c.maxBytesLimit)
+		f.Flag("max-bytes", "Maximum bytes to keep").StringVar(&c.maxBytesLimit)
```

Comment:
Suspect you only really needed to set this to BytesVar() and it would have mostly solved this